xds/src/main/java/io/grpc/xds/internal/rbac/engine/AuthorizationEngineInterface.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2021 The gRPC Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.grpc.xds.internal.rbac.engine;
```

Comment:
We may need to figure out how the package should be organized.

Reply:
The existing code for `internal.rbac.engine` could probably be deleted. It was dead code left over by security team's intern last summer. If I remember correctly, security team has changed the design and this is no longer needed.

/cc @ZhenLian for confirmation.

Reply:
likely ok to throw away, this is what I heard from ejona@.
I just leave it there for a little longer as long as it is not on our way. I didn't know the story about `internal` directory, but I guess this rbac engine is not google internal but for grpc xds itself, so I moved it to io.grpc.xds.